includes/revisions-control/README.md

```diff
@@ -0,0 +1,46 @@
+# Revisions Control
+
+This feature allows sites to limit the number of revisions kept for each post.
+
+## How it works
+
+Everytime a post is saved, WordPress check the maximum number of revisions should be kept in the database. By default, there's no limit.
+
+When you enable this feature, you will set the max number of revisions you want to preserve in the database. If there are more than the set maximum, older revisions will be deleted.
+
+This feature also takes into consideration the age of the revision. By default, a revisions will be deleted only if it is more than 1 week old. So even if the number of existing revisions exceeds the maximum number set, revisions will not be deleted if they are not old enough.
+
+See bellow how to change these settings.
+
+## Usage
+
+There are two ways to enable it.
+
+### Using a Constant
+
+Add this to your `wp-config.php`:
+
+```php
+define( 'NEWSPACK_LIMIT_REVISIONS_NUMBER', 30 );
+```
+
+Replace `30` with the maximum number of revisions you want to keep.
+
+When using the constant, the minimum age a revision must have to be deleted will default to 1 week.
+
+### Using an option
```

Comment:
I did this because at some point in the discussion it was mentioned that it would be nice to have a UI for that.

In any case, let's see what happens and what people need. We can add the filters later.